src/router/index.js

```diff
@@ -0,0 +1,80 @@
+import {RegisterPage} from "../pages/Register";
+import {ErrorPage} from "../pages/Error";
+import {LoginPage} from "../pages/Login";
+import {BaseAccountPage, EditedAccountPage} from "../pages/Account";
+import {ChatPage} from "../pages/Chat";
+
+function findRouteByUrl(url) {
+    let route = routes.filter((i) => i.url === url);
+    route = route[0];
```

Comment:
```suggestion
    let route = routes.find((i) => i.url === url);
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find